### PR TITLE
update for v7.6.1

### PR DESCRIPTION
**Pull request by brendanf**
Checklist
* [X] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dpryan79** [comment]
@conda-forge-admin please rerender

**Comment by dbast** [comment]
Newest cran version seems to be 7.5.2.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,6 +36,7 @@ requirements:
     - r-rlang >=0.2.0
     - r-storr >=1.1.0
     - r-txtq >=0.1.3
+    - r-knitr
```
Comment (dbast) [inline recipe/meta.yaml]: ~~This most probably needs to be also added to the run section.~~ Hm, it is not listed as imports...

Comment (dpryan79) [reply recipe/meta.yaml]: It's just a "suggests", so should be removed.

**Review by wlandau — APPROVED** [review]
I glanced at the changes, and besides https://github.com/conda-forge/r-drake-feedstock/pull/3/files#r314958861, I do not see anything else that needs fixing.

One note: I have not submitted version 7.6.0 to CRAN yet. (I cannot do so until the CRAN maintainers come back from vacation on Monday.)

**Comment by brendanf** [comment]
I'll wait for the new CRAN version to be ready (in the next few days I guess) and then remove `r-knitr` from `meta.yaml` to trigger the checks again.

File: recipe/meta.yaml (hunk 1 above)
Comment (brendanf) [reply recipe/meta.yaml]: Removed it.

**Pull request merged**